cve_bin_tool/version_scanner.py

```diff
@@ -33,15 +36,16 @@ class VersionScanner:
     """ "Scans files for CVEs using CVE checkers"""
 
     CHECKER_ENTRYPOINT = "cve_bin_tool.checker"
+    infogen = Generator[Tuple[Union[ProductInfo, None], str], None, None]
 
     def __init__(
         self,
-        should_extract=False,
-        exclude_folders=[],
-        checkers=None,
-        logger=None,
-        error_mode=ErrorMode.TruncTrace,
-        score=0,
+        should_extract: bool = False,
+        exclude_folders: list[str] = [],
+        checkers: dict[str, type[Checker]] | None = None,
```

Comment:
Because its the `Checker` subclass itself (uninstantiated) of type `CheckerMetaClass`.

https://github.com/intel/cve-bin-tool/pull/1581/files#diff-26f9fbe33c176687afb0a76b2e9bf31984528498fc2538ea95f3080f2c36bdeeR353-R354